src/main/java/adris/altoclef/tasks/speedrun/BeatMinecraft2Task.java

```diff
@@ -577,9 +577,12 @@ private Task getEyesOfEnderTask(AltoClef mod, int targetEyes) {
         int blazeRodTarget = (int)Math.ceil(((double)targetEyes - eyeCount - blazePowderCount) / 2.0);
         int enderPearlTarget = targetEyes - eyeCount;
         boolean needsBlazeRods = blazeRodCount < blazeRodTarget;
-        boolean needsBlazePowder = eyeCount + blazePowderCount < targetEyes;
+        boolean needsBlazePowder =  eyeCount + blazePowderCount < targetEyes; // when this is wrong, we get stuck in an ender eye loop
```

Comment:
I will adjust the comments after the merge, this is to help explain.

Ender eye fix